vue-begin/src/components/Employees.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+  <div>
+    <h1> {{ msg }}</h1>
+
+    <h2>Add new employee</h2>
+    <v-flex class="text-xs-center py-3">
+      <v-btn @click="showDialog(true,employee)" color="primary">
+        <v-icon>add</v-icon> Add Employee
+      </v-btn>
+    </v-flex>
+
+    <h2>Display all employees</h2>
+    <v-list three-line>
+      <v-list-tile v-for="employee in paginatedEmployees"
+        class="employees"
+        v-bind:key="employee.id"
```

Comment:
You can leave a shorthand version: `:key="employee.id"`

Reply:
ok